source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,229 @@
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+
+To match multiple terms in your text search, separate each term
+with spaces in the ``Filters.text()`` builder method. When you pass the 
+text search query returned by the builder method to the
+``find()`` method, it returns documents that match any of the terms. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+terms "fate" or "7".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin multipleTermExample
+   :end-before: end multipleTermExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Search Text by a Phrase
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Pass a phrase with escaped quotes to the ``Filters.text()`` method to
+specify the phrase in your text search. If you don't add escaped quotes,
+the method runs a :ref:`term search <term_search>`. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+phrase "fate of the furious". 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin phraseExample
+   :end-before: end phraseExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+ 
+Search Text with Terms Excluded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For each term you want to exclude from your text search, prefix the term
+with a minus sign in the string that you pass to the ``Filters.text()``
+builder method. 
+
+Each document matched does not contain the excluded term in
+your text index field. 
+
+.. note::
+
+   You must have at least one text search term if you want to
+   exclude terms from your search. 
+
+Example 
+```````
+
+The following example runs a text search on the documents in the
+``fast_and_furious_movies`` collection for titles that contain the
+term "furious", but do not contain the term "fast".
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin negateExample
+   :end-before: end negateExample
+
+The following shows the output of the preceding code:
+
+.. code-block:: json
+   :copyable: false
+
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+For more information on text indexes, see our Server Manual Entry on :manual:`Text
+Indexes </core/index-text/>`. 
```

Comment:
I'm not sure it has anything to do specifically with the example of excluding a term. Wouldn't it make more sense to put in the Text Indexes section of this page next to the link to the Indexes guide?

Reply:
It does, but the Indexes guide is linked in the Text Index section -- so should the same link be listed twice?